src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -0,0 +1,17 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class ColorSupplier {
+    private static final int NUMBER_OF_COLORS = Color.values().length;
+    private Random random = new Random();
+
+    public ColorSupplier() {
+        random = new Random();
+    }
```

Comment:
You already created `Random` instance, no need to create another object in constructor. Remove constructor.